### DISCUSSION TITLE
Markdown rendering bug - firefox

### DISCUSSION THREAD

**Discussion by BSoDium** [community/community#19112] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, there seems to be a bug related to markdown rendering, only visible on Mozilla Firefox:
![image](https://user-images.githubusercontent.com/46868627/174914715-addce098-c0ad-4b00-add9-55e567cc7b8a.png)
The images in my personal readme ([BSoDium/readme.md](https://github.com/BSoDium)) do not appear correctly, however, they do render perfectly on Google Chrome, which is odd.
Thanks in advance.

**Comment by melvincwng** [comment, marked as answer]
Hi @BSoDium, for bugs, please kindly raise a ticket to [GitHub Bug Report over here](https://support.github.com/contact/bug-report).

You would usually get a response within the next few working days.

Hope this clarifies.

**Reply by melvincwng** [reply]
@BSoDium, if you find this reply useful, please kindly 'Mark as answer' so that others are aware this issue has been addressed.

Thanks, and have a good day ahead 😄

**Reply by BSoDium** [reply]
I have successfully created a ticket, thank you for your help !
